src/content/guides/asset-management.md

```diff
@@ -512,14 +513,15 @@ The coolest part of everything mentioned above, is that loading assets this way
 + |  |  |– img.png
 ```
 
-This setup makes your code a lot more portable as everything that is closely coupled now lives together. Let's say you want to use `/my-component` in another project, simply copy or move it into the `/components` directory over there. As long as you've installed any _external dependencies_ and your _configuration has the same loaders_ defined, you should be good to go.
+This setup makes our code a lot more portable as everything that is closely coupled is now stored together. Let's say we want to use `/my-component` in another project, copy or move it into the `/components` directory. As long as we've installed any _external dependencies_ and our _configuration has the same loaders_ defined, we should be good to go.
```

Comment:
The "simply" here actually has a function, so I would recommend replacing it with something like "we can" or "we should".